internal/services/aws_iam_service.go

```diff
@@ -0,0 +1,58 @@
+package services
+
+import (
+	"net/http"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/render"
+)
+
+func ValidatePermissions(w http.ResponseWriter, r *http.Request) {
```

Comment:
This function currently only lists attached role policies, but the idea is to do the rest for validation here. I do not expect we would need the endpoint to use only ListAttachedPolicies, but I left the function call there so that we'd see the outcome.